src/mock/ovirtapi.mock.js

```diff
@@ -0,0 +1,210 @@
+import $ from 'jquery'
+
+import { logDebug } from '../helpers'
+import { Exception } from '../exceptions'
+import Selectors from '../selectors'
+import AppConfiguration from '../config'
+import { vms, disks, api } from './data.mock'
+
+let OvirtApi = {}
+OvirtApi = {
+  // ----
+  /**
+   * @param vm - Single entry from oVirt REST /api/vms
+   * @returns {} - Internal representation of a VM
+   */
+  vmToInternal ({ vm }) {
```

Comment:
Can we avoid copy&paste as much as possible and "reuse" original OvirtApi?
This mock js file should contain more or less just the mock calls.

This requirement is tricky, I know, but fundamental.